octobot_services/notifier/abstract_notifier.py

```diff
@@ -103,9 +103,10 @@ async def _handle_order_notification(self, dict_order, linked_notification, orde
             self.previous_notifications_by_identifier[order_identifier] = notification
         else:
             is_simulated = trading_api.is_trader_simulated(exchange_manager)
-            if order_status is trading_enums.OrderStatus.CANCELED or \
+            if order_status is trading_enums.OrderStatus.EXPIRED or \
+                order_status is trading_enums.OrderStatus.CANCELED or \
                     (order_status is trading_enums.OrderStatus.CLOSED
-                     and dict_order[trading_enums.ExchangeConstantsOrderColumns.FILLED.value] == 0):
+                     and dict_order[trading_enums.ExchangeConstantsOrderColumns.FILLED_AMOUNT.value] == 0):
```

Comment:
Can you restore this change so we can merge before merging trading please?

Reply:
Done, I moved it to a different PR